rero_ils/modules/ext.py

```diff
@@ -152,7 +153,8 @@ def register_signals(self, app):
         after_record_insert.connect(create_subscription_patron_transaction)
         after_record_update.connect(create_subscription_patron_transaction)
         
-        after_record_insert.connect(create_operation_log_record)
+        after_record_insert.connect(operation_log_record_create)
+        after_record_update.connect(operation_log_record_update)
```

Comment:
create an issue/tasks to implement 'delete' operation with the signal 'before_record_delele' ?

Reply:
The current Virtua `039` implementation captures only add/update, I will pass the info to @pronguen to decide issue/ticket/task or US.

Reply:
To be done later. It requires a deeper analysis and also to keep the records with a status `deleted`.